packages/desktop-client/src/components/reports/spreadsheets/spending-spreadsheet.ts

```diff
@@ -167,6 +200,7 @@ export function createSpendingSpreadsheet({
         average: integerToAmount(averageSum) / monthCount,
         thisMonth: dayData[3].cumulative,
         lastMonth: dayData[2].cumulative,
+        lastYear: dayData[0].cumulative,
```

Comment:
Is this being used anywhere? I'd assume it should be used here:
https://github.com/actualbudget/actual/blob/08aff05a06f4bca4344f939324080e1fa6511b62/packages/desktop-client/src/components/reports/reports/Spending.tsx#L198